tests/wpunit/TEC/Events/Admin/Notice/RESTAPITest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * REST API Notice test.
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Tests\Events\Admin\Notice;
+
+use Codeception\TestCase\WPTestCase;
+use TEC\Events\Admin\Notice\Rest_Api;
+use Tribe\Tests\Traits\With_Uopz;
+use WP_Error;
+
+/**
+ * Class RESTAPITest
+ *
+ * @since TBD
+ */
+class RESTAPITest extends WPTestCase {
+
+	use With_Uopz;
+
+	/**
+	 * Clean up after this test class finishes.
+	 *
+	 * @return void
+	 */
+	public function wpTearDownAfterClass() {
+		remove_all_filters( 'tec_events_site_is_development_mode' );
+		remove_all_filters( 'tec_events_rest_api_response_blocked_due_to_timeout' );
+	}
```

Comment:
I'd prefer if you used `@after` or `@afterClass` instead of the `wpTearDownAfterClass` method. I have discussed with Luca that the annotation versions are better supported in between version of PHPUnit.